csrc/executor.cpp

```diff
@@ -681,18 +992,25 @@ std::vector<at::Tensor> allocOutputs(
       // integration, see step 1 - note [trivial forwarding]
       const auto tensor_options =
           at::TensorOptions().dtype(at::kFloat).device(device);
-      outputs.emplace_back(at::empty({0}, tensor_options));
+      outputs.emplace_back(at::empty(
```

Comment:
Why does this need to be changed this way? Isn't this just a special WAR for aliased inputs/outputs?

Reply:
Yes, it is, but still, `transformOutputFromAllocationToRFactor` will run on these tensors, and having a wrong dimensionality will cause assertion failure in this function.

Reply:
I actually thought so, but isn't it only called in the else case at line 1012?

Reply:
Hmmm, good question. I am pretty sure that I do see failures, maybe that was only needed when I was playing with it, and no longer needed now. Will check.

Reply:
You are right. It is no longer needed.